java/bulk-export/bulk-export-core/src/test/java/sleeper/bulkexport/model/BulkExportLeafPartitionQuerySerDeTest.java

```diff
@@ -0,0 +1,356 @@
+/*
+ * Copyright 2022-2024 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.bulkexport.model;
+
+import org.junit.jupiter.api.Test;
+
+import sleeper.core.properties.instance.InstanceProperties;
+import sleeper.core.properties.table.TableProperties;
+import sleeper.core.properties.testutils.FixedTablePropertiesProvider;
+import sleeper.core.range.Range.RangeFactory;
+import sleeper.core.range.Region;
+import sleeper.core.schema.Field;
+import sleeper.core.schema.Schema;
+import sleeper.core.schema.type.IntType;
+import sleeper.core.schema.type.StringType;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static sleeper.core.properties.table.TableProperty.TABLE_ID;
+import static sleeper.core.properties.table.TableProperty.TABLE_NAME;
+import static sleeper.core.properties.testutils.InstancePropertiesTestHelper.createTestInstanceProperties;
+import static sleeper.core.properties.testutils.TablePropertiesTestHelper.createTestTablePropertiesWithNoSchema;
+
+public class BulkExportLeafPartitionQuerySerDeTest {
+    private final Field field = new Field("key", new IntType());
+    private final Schema schema = Schema.builder()
+            .rowKeyFields(field)
+            .valueFields(new Field("value1", new StringType()), new Field("value2", new StringType()))
+            .build();
+    private final InstanceProperties instanceProperties = createTestInstanceProperties();
+    private final TableProperties tableProperties = createTestTablePropertiesWithNoSchema(instanceProperties);
+    private final String tableId = tableProperties.get(TABLE_ID);
+    private final String exportId = UUID.randomUUID().toString();
+    private final String subExportId = UUID.randomUUID().toString();
+    private final String leafPartitionId = UUID.randomUUID().toString();
```

Comment:
I think these UUIDs make the test harder to read. Could we please set the test data directly in each test, so we can see the real values in the setup and in the JSON?

Reply:
These have been removed